src/services/resourceService.ts

```diff
@@ -14,11 +14,9 @@ export class ResourceService extends BaseService {
   public async deployResourceGroup() {
     this.log(`Creating resource group: ${this.resourceGroup}`);
 
-    const groupParameters = {
-      location: this.serverless.service.provider["location"]
-    };
-
-    return await this.resourceClient.resourceGroups.createOrUpdate(this.resourceGroup, groupParameters);
+    return await this.resourceClient.resourceGroups.createOrUpdate(this.resourceGroup, {
+      location: this.getRegion(),
```

Comment:
rename `location` to `region` to be consistent

Reply:
Can't change it here - this is the call to the ARM sdk and it expects `location`.